app/facades/station_facade.rb

```diff
@@ -0,0 +1,10 @@
+class StationFacade
+  require "./app/services/station_service.rb"
```

Comment:
You shouldn't need to require any files. Everything in `/app` is loaded up and available in your rails application by default.